### PR TITLE
Blog post about WASI policies

### PR DESCRIPTION
**Pull request by flavio**
Fixes issue #190

Not ready for review, I just want to try the new netlify integration I've enabled.

**Update:** it works :sunglasses: https://deploy-preview-191--moonlit-trifle-c6359f.netlify.app/blog/2023/09/wasi-policies/

### CODE REVIEW

**Comment by flavio** [comment]
@jhkrug could you please take a look at this blog post? This is a technical/nerdy topic, I wonder if I'm using the right approach... Is it too long, is it clear enough?

**Review by jhkrug — APPROVED** [review]
Hi @flavio. Reads great. A few suggested changes to consider. I think there is only one of them which needs particular care as it might change the meaning (which I'm technically unsure of), lines 30-35. 

Happy to discuss any of this in more detail as necessary. Thanks.

File: content/blog/2023/09/wasi-policies.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ones produced by the TinyGo compiler).
+
+## Go and server-side Wasm
+
+The Go 1.21 compiler, recently released, introduces proper support for WebAssembly modules that can
+be run on the server side. Before this release, the official compiler was able to produce only
+WebAssembly modules that could be run by a web browser.
+While [TinyGo](https://tinygo.org/) does an excellent job of producing this
+kind of WebAssembly module, there are still some complex Go codebases that cannot be compiled
+by it.
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2023/09/wasi-policies.md:31]: > [TinyGo](https://tinygo.org/) does an excellent job of producing server side WebAssembly modules but there are still some complex Go codebases that cannot be compiled by it.

This is how it should be fixed, I'm going to do that manually

File: content/blog/2023/09/wasi-policies.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+To produce a Kubewarden WASI policy, the policy author writes a traditional CLI program. The program will
+obtain the evaluation payload from the `STDIN` and will write the evaluation response on the `STDERR`.
+This completely removes the dependency on waPC.
+
+## A concrete example
+
+Kubewarden aims to become the universal policy framework for Kubernetes. Meaning, we don't want to prescribe
+how policy authors should write their policies. On the opposite, we intend to provide a flexible way to
+express rules and constraints.
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2023/09/wasi-policies.md]: So much better, thanks!

File: content/blog/2023/09/wasi-policies.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2023/09/wasi-policies.md]: fixed with e4e45c639b6f83935920769abb9da3305b20c37c

**Comment by flavio** [comment]
@jhkrug thanks for the review. I've applied all your suggestions

**Review by jvanz — APPROVED** [review]

**Pull request merged**